styles.css

```diff
@@ -0,0 +1,153 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+  --primary-color : #c2004b;
+  --secondary-color : #eb679a;
+  --light-color : #ffeff5;
+  --dark-color : #72022d;
+
+  --alert-primary : #cfe2ff;
+  --alert-primary-color : #084298;
+  --alert-success : #d1e7dd;
+  --alert-success-color : #0f5132;
+  --alert-warning : #fff3cd;
+  --alert-warning-color : saddlebrown;
+  --alert-link : #cff4fc;
+  --alert-link-color : #055160;
+  --alert-similar-to-theme : #ffd0e1;
+}
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto;
+}
+::-webkit-scrollbar {
+  width: 0px;
+}
+html {
+  width: 100%;
+  height: 100%;
+}
+body {
+  display: flex;
```

Comment:
You can consider using flex-col utility class instead, since you'll be using it many times

Reply:
yes, good idea. Thanks